xarray/test/test_plot.py

```diff
@@ -112,6 +112,17 @@ def test__infer_interval_breaks(self):
         self.assertArrayEqual(pd.date_range('20000101', periods=4) - np.timedelta64(12, 'h'),
                               _infer_interval_breaks(pd.date_range('20000101', periods=3)))
 
+        # make a bounded 2D array that we will center and re-infer
+        xref, yref = np.meshgrid(np.arange(6), np.arange(5))
+        cx = (xref[1:, 1:] + xref[:-1, :-1]) / 2
```

Comment:
On Python 2, I think this will still have integer dtype. Can you divide by `2.0` instead?